dotCMS/src/main/java/com/dotmarketing/portlets/workflows/util/WorkflowImportExportUtil.java

```diff
@@ -204,6 +209,23 @@ public void importWorkflowExport(final WorkflowSchemeImportExportObject importer
 		}
 	}
 
+	private Role getAnonRole () {
+		return Try.of(()->APILocator.getRoleAPI().loadCMSAnonymousRole()).getOrNull();
```

Comment:
Maybe use`getOrThrow` and throw a new DotRuntimeException rather than `getOrNull()`